api/libproject/devstart.rb

```diff
@@ -634,21 +645,31 @@ def run_local_all_migrations()
 
 Common.register_command({
   :invocation => "run-local-all-migrations",
-  :description => "Runs local data/schema migrations for cdr/workbench schemas.",
+  :description => "Runs local data/schema migrations for the cdr and workbench schemas.",
   :fn => ->() { run_local_all_migrations() }
 })
 
-
 def run_local_data_migrations()
   init_new_cdr_db %W{--cdr-db-name cdr --run-list data --context local}
 end
 
 Common.register_command({
   :invocation => "run-local-data-migrations",
-  :description => "Runs local data migrations for cdr/workbench schemas.",
+  :description => "Runs local data migrations for the cdr schema.",
   :fn => ->() { run_local_data_migrations() }
 })
 
+def run_local_rw_migrations()
+  common = Common.new
+  common.run_inline %W{docker-compose run db-scripts ./run-migrations.sh main}
+end
+
+Common.register_command({
+  :invocation => "run-local-rw-migrations",
```

Comment:
To be clear, `dev-up` already does just this. It might be nice to have this option without a full dev-up, but under normal circumstances in the dev flow you shouldn't need to use `run-local-all-migrations` unless you're developing cohort builder.

Reply:
Right: this is only to speed up the non-dev-up by making it possible to run workbench liquibase updates without waiting ~6 minutes for an unnecessary CDR migration